### DISCUSSION TITLE
i want worldmap mods 我想要大地图/小地图模组

### DISCUSSION THREAD

**Discussion by Rad586** [Rad586/Forget-Me-Not#55] [Q&A] [answered]

**Comment by Rad586** [comment, marked as answer]
SEARCH and delete incompat_mods.js
搜索 并删除incompat_mods.js即可

edit: not a feature in 2.4.0
编辑：2.4.0已移除此特性

**Reply by LinXin-L** [reply]
为什么这个mod在文件夹里没有，但是装了xaero的地图模组说不兼容请删除，请问要怎么做

**Reply by Rad586** [reply]
搜索这个文件即可

**Reply by LinXin-L** [reply]
![1313](https://github.com/Rad586/Forget-Me-Not/assets/162077536/fcf5a10d-d390-4238-adb4-fd1782e6be08)
感谢回复，但是我在mods文件夹搜索并没有搜索到，而且我在forget me not大文件夹界面也没有搜索到。
在游戏里使用地图模组只是屏幕上有模组不兼容请删除的红色文字，目前还没有发现问题。这个incompat模组搜索不到还有什么模组是冲突相关的吗？感谢楼主

**Reply by Rad586** [reply]
![image](https://github.com/Rad586/Forget-Me-Not/assets/99306685/f0732fc0-906e-4ed1-b892-bd3edf3235e6)
这里搜

**Reply by LinXin-L** [reply]
感谢楼主，是这里面,已解决
